pages/artist/_itemId/index.vue

```diff
@@ -12,44 +12,42 @@
           />
         </v-col>
         <v-col cols="12" sm="7">
-          <div class="ml-sm-4 d-flex flex-column">
-            <div
-              class="text-subtitle-1 text--secondary font-weight-medium text-capitalize"
-            >
-              {{ $t('artist') }}
+          <v-row justify="space-between">
+            <div class="ml-sm-4 d-flex flex-column">
+              <div
+                class="text-subtitle-1 text--secondary font-weight-medium text-capitalize"
+              >
+                {{ $t('artist') }}
+              </div>
+              <h1 class="text-h4 text-md-h2 font-weight-light">
+                {{ item.Name }}
+              </h1>
             </div>
-            <h1 class="text-h4 text-md-h2 font-weight-light">
-              {{ item.Name }}
-            </h1>
-          </div>
+            <div class="d-flex align-center">
+              <item-menu :item="item" />
+              <like-button :item="item" />
+            </div>
+          </v-row>
         </v-col>
       </v-row>
       <v-row>
         <v-col>
           <v-tabs v-model="activeTab" background-color="transparent">
-            <v-tab :key="0">{{ $t('overview') }}</v-tab>
-            <v-tab :key="1">{{ $t('about') }}</v-tab>
+            <v-tab :key="0">{{ $t('item.artist.discography') }}</v-tab>
+            <v-tab :key="1">{{ $t('item.artist.information') }}</v-tab>
```

Comment:
Do we need to remove `overview` and `about` or are they used elsewhere

Reply:
Only ``about``, ``overview`` it's used. I need to clean this up furthermore so all the item pages and elements are in their own subkey, but that's for a further PR.

Reply:
I've checked, about can be removed. Overview is used elsewhere